template/tests/SERVICE_NAME.service.spec.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Entry point for unit test.
+ * Uses the moleculer microservices framework.
+ *
+ * Copyright Byte Technology 2020. All rights reserved.
+ */
+
+import { Service as MoleculerService } from 'moleculer';
+
+import { startService, stopService, broker } from '../src/lib/service.broker';
+import { resetServiceDB } from './utils';
+
+describe('{{capitalizedServiceName}} unit tests', () => {
+  let service: MoleculerService;
+
+  beforeAll(async done => {
+    service = await startService();
+    done();
+  });
```

Comment:
add `timeout` to prevent async errors due to `beforeAll` timing out before the service has started.

Reply:
Global Jest timeouts are already set in 2 places.
1) `package.json => jest.testTimeout`
2) `tests/setup.ts` - starts before all tests - you can override there `testTimeout` from `package.json` by setting `JEST_TIMEOUT` env var.
Execution of `tests/setup.ts` is guaranteed by `package.json => jest.setupFilesAfterEnv`